backend/src/main/java/ddangkong/service/room/balance/roomvote/RoomBalanceVoteService.java

```diff
@@ -51,4 +52,11 @@ public boolean isAllMemberVoted(RoomMembers roomMembers, BalanceOptions balanceO
     public List<RoomBalanceVote> getVotesInRoom(RoomMembers roomMembers, BalanceOption balanceOption) {
         return roomVoteRepository.findByMemberInAndBalanceOption(roomMembers.getMembers(), balanceOption);
     }
+
+    public List<RoomBalanceVote> findRoomVotesByBalanceOptionsWithoutMember(List<BalanceOption> memberRoomVoteOptions,
+                                                                            Room room,
+                                                                            Member member) {
+        return roomVoteRepository.findRoomBalanceVotesByBalanceOptionsAndRoomWithoutMember(memberRoomVoteOptions, room,
+                member);
+    }
```

Comment:
사소한 의견) 클래스 명 등으로 명시된 정보는 생략해도 될 것 같아요~
```suggestion
    public List<RoomBalanceVote> findVotesByOptionsWithoutMember(List<BalanceOption> memberRoomVoteOptions,
                                                                            Room room,
                                                                            Member member) {
        return roomVoteRepository.findRoomBalanceVotesByBalanceOptionsAndRoomWithoutMember(memberRoomVoteOptions, room,
                member);
    }
```

Reply:
동의.